apps/desktop/src/renderer/ui/primitives/Toast.module.css

```
@@ -0,0 +1,141 @@
+  background: var(--accent);
+}
+
+.warning .dot {
+  background: var(--warning);
+}
+
+/* ── Message ───────────────────────────────────────────────────────────────── */
+
+.message {
+  flex: 1;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)